Assets/InputSystem.Extras/DualShock.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
+guid: f14c62904b464a64a6dce94fc2ba0d9a
+folderAsset: yes
+DefaultImporter:
+  externalObjects: {}
+  userData: 
+  assetBundleName: 
+  assetBundleVariant: 
```

Comment:
When committing with git, please stage only the files that you actually want to touch instead of staging every modified file.

Reply:
This one here adds a file back that was moved elsewhere.